capstone/src/main/webapp/createStudySet.html

```diff
@@ -1,21 +1,128 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>Studentia-Create Study Set</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css" />
     <!--Import Google Icon Font-->
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
     <!--Import materialize.css-->
-    <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
+    <link
+      type="text/css"
+      rel="stylesheet"
+      href="css/materialize.min.css"
+      media="screen,projection"
+    />
 
     <!--Let browser know website is optimized for mobile-->
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
-     <h3>Create Study Set</h3>
-     <!--JavaScript at end of body for optimized loading-->
-     <script type="text/javascript" src="js/materialize.min.js"></script>
+    <nav class="nav-wrapper">
+      <a href="#" class="brand-logo">Studentia</a>
+      <a href="#" class="sidenav-trigger" data-target="slide_out"
+        ><i class="material-icons">menu</i></a
+      >
+      <ul id="nav-mobile" class="right hide-on-med-and-down">
+        <li><a href="findStudySet.html">Search</a></li>
+        <li><a href="index.html">Home</a></li>
+      </ul>
+    </nav>
+
+    <div class="row">
+      <div class="col s4 left-align teal lighten-4 z-depth-3 create-box">
+        <p class="flow-text white-text">First Step to Create Study Set</p>
+
+        <!-- This is will be replaced in a later PR -->
+        <div class="input-field col s12">
+          <input type="text" id="user_id" name="user_id" required/>
```

Comment:
IDs should all use hyphens rather than underscores: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters. It's fine for name to use underscores if that's what your server expects in the request body.